test/integration/scheduler/taint_test.go

```diff
@@ -79,7 +79,11 @@ func TestTaintNodeByCondition(t *testing.T) {
 	defer testutils.CleanupTest(t, testCtx)
 
 	cs := testCtx.ClientSet
-	informers := testCtx.InformerFactory
+	// don't share informer factories between scheduler and
```

Comment:
https://github.com/kubernetes/kubernetes/pull/90865 is trying to address this issue